app/cdap/components/ConnectionList/Components/HeaderCustomTooltipLabel/index.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Typography } from '@material-ui/core';
+import { IHeaderCustomTooltipLabelProps } from 'components/ConnectionList/types';
+import React from 'react';
+
+export default function({ index, headersRefs, filteredData }: IHeaderCustomTooltipLabelProps) {
+  return (
+    <Typography
```

Comment:
plz check if this component can be a styled variation of renderlabel
we need to reuse as much as decently possible

thank you

Reply:
Done